apps/evm/src/pages/Vai/Repay/index.tsx

```diff
@@ -90,16 +89,16 @@ export const Repay: React.FC = () => {
 
   const { mutateAsync: repayVai } = useRepayVai();
 
-  const { data: getVaiRepayAprData } = useGetVaiRepayApr();
-
-  const readableBorrowApr = formatPercentageToReadableValue(getVaiRepayAprData?.repayAprPercentage);
-
   const { data: getPoolData, isLoading: isGetPoolDataLoading } = useGetPool({
     poolComptrollerAddress: chain.corePoolComptrollerContractAddress,
     accountAddress,
   });
+  const legacyPool = getPoolData?.pool;
+  const userVaiBorrowBalanceTokens = legacyPool?.vai?.userBorrowBalanceTokens;
 
-  const userVaiBorrowBalanceTokens = getPoolData?.pool.userVaiBorrowBalanceTokens;
+  console.log(userVaiBorrowBalanceTokens?.toFixed());
```

Comment:
Stray log

Reply:
Always and ever... Thanks, it's removed.